spec/src/main/asciidoc/chapters/language/statements.adoc

```diff
@@ -8,18 +8,44 @@ As discussed in <<algebra>>, there is a logical ordering of clauses, reflecting
 
 1. `from`
 2. `where`,
-3. `group`
-4. `having`,
+3. `#group#`
+4. `#having#`,
 5. `select` or `set`,
 6. `order`.
 
 The interpretation and effect of each clause in this list is influenced by clauses occurring earlier in the list, but not by clauses occurring later in the list.
 
 ==== From clause
 
-The `from` clause, with syntax given by `from_clause`, specifies an _entity name_ which identifies the queried entity. Path expressions occurring in later clauses are interpreted with respect to this entity. That is, the first element of each path expression in the query must be a persistent attribute of the entity named in the `from` clause. The entity name is a Java identifier, usually the unqualified name of the entity class, as specified in <<Entity names>>.
+In the core language, the `from` clause, with syntax given by `from_clause`, specifies an _entity name_ identifying the root entity of the query, as defined in <<root-entity>>.
+This entity is always assigned the implicit alias `this`.
 
-NOTE: The syntax of the `update` statement is irregular, with the `from` keyword implied. That is, the syntax _should_ be `update from Entity`, but for historical reasons it is simply `update Entity`.
+In the core language, path expressions occurring in later clauses are interpreted with respect to this entity. That is, the first element of each path expression in the query must be a persistent attribute of the entity named in the `from` clause. The entity name is a Java identifier, usually the unqualified name of the entity class, as specified in <<entities-embeddables>>.
+
+NOTE: The core language does not have joins, and so the `from` clause is always trivial.
```

Comment:
Good  point❤️ 

Eventually, we need to create a section to explain what each one can do.